### ISSUE TITLE
Bug: no-invalid-properties false positive for relative colors

### ISSUE THREAD

**Issue by SwetaTanwar** [eslint/css#246]
### Environment

main branch

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
/* eslint css/no-invalid-properties: "error" */
```
</details>

```css
a {
   background: hwb(from hsl(0 100% 50%) h 30% b); // Error
   color: hsl(from #123456 h s calc(l + 20)) // Error
   color: hsla(from #123456 h s calc(l + 20)) // Error
   color: hwb(from green h w b / 0.5); // Error
   color: lch(from green l c h / 0.5); // Error
}
```

### What did you expect to happen?

All are valid syntax, there should be no error.

### What actually happened?

<img width="1410" height="142" alt="Image" src="https://github.com/user-attachments/assets/242b80ba-ea8e-4093-b253-a7516f74a326" />

### Link to Minimal Reproducible Example

Provided above

### Participation

- [x] I am willing to submit a pull request for this issue.

### Additional comments

I noticed this while working on https://github.com/eslint/css/issues/235, If the team aligns I'll gladly send a PR for fix.
